Matrixfile

```diff
@@ -134,9 +134,11 @@
   'lograge' => {
     'activesupport' => '✅ 2.5 / ✅ 2.6 / ✅ 2.7 / ✅ 3.0 / ✅ 3.1 / ✅ 3.2 / ✅ 3.3 / ✅ 3.4 / ✅ 3.5 / ✅ jruby'
   },
+  # MongoDB tests excluded for Ruby 3.5 due to bson gem native extension compilation failures
+  # with the Ruby 3.5.0-preview1 image. TODO: Ruby 3.5 - Re-enable after investigation.
```

Comment:
I think it's OK with a recent `bson` gem, but minimum `mongodb` version might need to be raised.